fairmq/plugins/PMIx/PMIx.hpp

```diff
@@ -47,172 +108,264 @@ struct rank
         local_node = PMIX_RANK_LOCAL_NODE
     };
 
-    explicit rank(pmix_rank_t r)
-        : m_value(r)
+    Rank(pmix_rank_t rank)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Struct 'Rank' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/app/FairRootGroup/FairMQ/pullRequest?prid=4639873)